www/source/docs/developing-packages.html.md.erb

```diff
@@ -87,6 +87,47 @@ All plans must have a `plan.sh` or `plan.ps1` at the root of the plan context. T
 
 When writing a plan, it's important to understand that you are defining both how the package is built and the actions Chef Habitat will take when the Supervisor starts and manages the child processes in the package. The following sections explain what you need to do for each phase.
 
+## <a name="plan-targets" id="plan-targets" data-magellan-target="plan-targets" type="anchor">Writing a Plan for Multiple Platform Targets</a>
+
+You may want to create a plan for an application that can run on multiple platform targets. You can create target specific folders beneath either the root of your project or a top level `habitat` folder. Then save the plan, hooks, and configuration templates specific to a single platform all inside of that target specific folder.
+
+For example, an application targeting linux, Linux kernel 2, and Windows may have the following structure:
+
+```
+app_root/
+├── x_86_64-linux/
+|   |   plan.sh
+|   └── hooks/
+|           run
+├── x_86_64-linux-kernel2/
+|   |   plan.sh
+|   └── hooks/
+|           run
+└── x_86-64-windows/
+    |   plan.ps1
+    └── hooks/
+            run
+```
+
+The build script will look for the base of your plan in the following locations:
+
+- `<app_root>/`
+- `<app_root>/habitat/`
+- `<app_root>/<target>/`
+- `<app_root>/habitat/<target>/`
```

Comment:
Again, I feel like it's a bit more natural and clear to list the locations in order of preference. At least, that's the order I would expect to read them in.

The fact that `<app_root>/habitat/` and `<app_root>/<target>` aren't swapped here is a little odd to me, in that it kind of turns the natural hierarchical nature of the filesystem inside-out.